storybook/src/pages/amopis/common/amopis.css

```diff
@@ -0,0 +1,62 @@
+.ams-docs-amopis-app-grid {
```

Comment:
I would love to build an example with existing components and present it to you and Mike.

Reply:
An example of what?

Reply:
An application without all the greys, blues and nested navigation. But this is a different discussion then this PR, sorry.